x/dymns/types/simple_concurrent_map.go

```diff
@@ -0,0 +1,47 @@
+package types
+
+import "sync"
+
```

Comment:
I guess you just had this, and the test, for experimentation/learning exercise?

Reply:
no, this is actually used for simple caching if a chain-id is a RollApp

Reply:
this caused AppHash on Playground so it was removed
cc @zale144